_summaries/hedman2021baking_2.md

```diff
@@ -13,7 +13,38 @@ TODO: Summarize the paper:
 * How well does the paper perform?
 * What interesting variants are explored?
 
+## Core Idea
+View synthesis is the task of given a set of images of a specific scene, generating a 3d representation of that scene. An innovative model in this task is NeRF, however this model is slow and intensive. This paper presents SNeRG, a model thats two orders of magnitude faster than NeRF. This is done by trading computation for storage through pre-training.
+
+<p align="center"> <img src="hedman2021baking_2_a.png" height="300"/> </p>
+
+## How This is Realized
+This is done by precomputing and storing ("baking") a trained NeRF into a Sparse Neural Radiance Grid which is represented as a sparce 3d voxel grid data structure.
+
+During training, the model uses a "deferred" NeRF model which computes environmental features but differs view-dependent features to render time. During testing, a method  of storing the model in a sparse 3D data structue is used which allows for quick speed ups when executing the model. From here diffuse color features volume densities and specular features are passed to generate alpha-composite colors and features. The features are then passed though a neural network along with the viewing direction to generate a specular color which is finally concatenated to the colors to generate an output.
```

Comment:
It is a good summary. However, I think it will be better if you can explain more about what kind of data will be stored in SNeRG, and what trick they use to avoid dense voxel representation. Maybe adding a little more interpretations for why SNeRG's rendering time is way shorter than NeRF's rendering time will be nice.

Reply:
It will also be nice if you can add some descriptions about tricks they used, such as compression, opacity regularitzation, and sparsity loss and visibility culling

Reply:
Done!